client/core/core.go

```diff
@@ -217,6 +217,7 @@ func (dc *dexConnection) marketMap() map[string]*Market {
 			EpochLen:        mkt.EpochLen,
 			StartEpoch:      mkt.StartEpoch,
 			MarketBuyBuffer: mkt.MarketBuyBuffer,
+			RecentMatches:   mkt.RecentMatches,
```

Comment:
Recent matches won't be be coming from a `'config'` response (in the `msgjson.Market`).  I wasn't sure the best place to keep the data, so I reviewed what we're doing for other `'orderbook'` and `'spots'` subscription notifications.

Currently we decided to use the `EpochReportNote`, which is part of the `orderbook` subscription, to communicate recent matches from the server in notifications.  Time-stamped recent matches for a market would be tracked by some new fields in `dexConnection` (e.g. the `spots` and `spotsMtx` fields), or possibly in the `bookie` for a given market (see how `(*bookie).logEpochReport(*msgjson.EpochReportNote)` already does this for this note but for the `Candle` data).

Now note how we send `BookUpdate`s to consumers like the frontend, with various "actions":
https://github.com/decred/dcrdex/blob/3a476f110fe1e295b20a400edb34f6e08fe1fc74/client/core/types.go#L570-L578

So, I imagine we'd follow that pattern.

Reply:
I am using now `epoch_report` to update recent matches